features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -92,6 +92,11 @@
         {% endif %}
     </ServiceProviders>
 
+    <LegacyFeatures>
+        <EnableOAuth1>{{legacy_feature.enable_oauth1}}</EnableOAuth1>
```

Comment:
shall we make this legacy_features.enable_oauth1? and the below one too

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2997/commits/30e3885b3cab329ad0b92e2fa02e7dbfaf77c413